components/carbon-transports/websocket/org.wso2.carbon.websocket.transport/src/main/java/org/wso2/carbon/websocket/transport/SubprotocolBuilderUtil.java

```diff
@@ -29,6 +29,7 @@
     private static String SYNAPSE_SUBPROTOCOL_PREFIX = "synapse(";
     private static String SYNAPSE_SUBPROTOCOL_SUFFIX = ")";
     private static String SYNAPSE_CONTENT_TYPE = "contentType=";
+    private static String SYNAPSE_CONTENT_TYPE_GRAPHQL_WS = "graphql-ws";
```

Comment:
This makes WS implementation aware of a sub-protocol called "graphql-ws" which is not correct.